Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -158,6 +161,10 @@ public struct LambdaContext: CustomDebugStringConvertible, Sendable {
     public var allocator: ByteBufferAllocator {
         self.storage.allocator
     }
+    
+    public var tasks: DetachedTasksContainer {
```

Comment:
I don't think we need to make the task container a property here. Instead we should add a `func detachedBackgroundTask(_ body: @escaping @Sendable () async -> ())` API here. Which then uses the task container internally.